react/src/components/Widget/Widget.tsx

```diff
@@ -372,29 +364,18 @@ export const Widget: React.FC<WidgetProps> = props => {
   };
 
   useEffect(() => {
-    const inSatoshis = getCurrencyObject(totalSatsReceived, 'SAT');
+    // const inSatoshis = getCurrencyObject(totalSatsReceived, 'SAT');
```

Comment:
Do we need to keep this?

Reply:
I wont tell anyone if you don't

(I fixed it in the next stacked branch, I'll see if i can do it here without fucking up git)